code/game/gamemodes/clock_cult/clock_scriptures/scripture_scripts.dm

```diff
@@ -70,6 +70,12 @@
 	quickbind = TRUE
 	quickbind_desc = "Creates a Vitality Matrix, which drains non-Servants on it to heal Servants that cross it."
 
+/datum/clockwork_scripture/create_object/vitality_matrix/check_special_requirements()
+	if(locate(object_path) in range(1, invoker))
```

Comment:
This isn't a checkerboard pattern, since you're using a square range instead of a diamond range. You should use circlerange or the range that's cardinal that I can never remember.

Reply:
I might have misphrased; it *is* intentional to prevent an adjacency checkboard pattern, so that people can go between the matrices without having to cross a bunch of them.